{{cookiecutter.project_slug}}/frontend/src/views/main/admin/EditUser.vue

```diff
@@ -1,145 +1,187 @@
                 <v-text-field
-                  :disabled="!setPassword"
-                  type="password"
-                  ref="password"
-                  label="Set Password"
-                  data-vv-name="password"
-                  data-vv-delay="100"
-                  v-validate="{required: setPassword}"
-                  v-model="password1"
-                  :error-messages="errors.first('password')"
-                >
-                </v-text-field>
+                  v-model="fullName"
+                  label="Full Name"
+                  :error-messages="errors[0]"
+                  required
+                ></v-text-field>
+              </validation-provider>
+
+              <!-- email -->
+              <validation-provider
+                v-slot="{ errors }"
+                rules="required|email"
+                name="E-mail"
+              >
                 <v-text-field
-                  v-show="setPassword"
-                  type="password"
-                  label="Confirm Password"
-                  data-vv-name="password_confirmation"
-                  data-vv-delay="100"
-                  data-vv-as="password"
-                  v-validate="{required: setPassword, confirmed: 'password'}"
-                  v-model="password2"
-                  :error-messages="errors.first('password_confirmation')"
-                >
-                </v-text-field>
-              </v-flex>
-            </v-layout>
-          </v-form>
-        </template>
-      </v-card-text>
-      <v-card-actions>
-        <v-spacer></v-spacer>
-        <v-btn @click="cancel">Cancel</v-btn>
-        <v-btn @click="reset">Reset</v-btn>
-        <v-btn
-          @click="submit"
-          :disabled="!valid"
-        >
-          Save
-        </v-btn>
-      </v-card-actions>
-    </v-card>
+                  v-model="email"
+                  label="E-mail"
+                  type="email"
+                  :error-messages="errors[0]"
+                  required
+                ></v-text-field>
+              </validation-provider>
+
+              <!-- is superuser -->
+              <div class="subheading secondary--text text--lighten-2">
+                User is superuser
+                <span v-if="isSuperuser">(currently is a superuser)</span
+                ><span v-else>(currently is not a superuser)</span>
+              </div>
+              <v-checkbox v-model="isSuperuser" label="Is Superuser"></v-checkbox>
+
+              <!-- is active -->
+              <div class="subheading secondary--text text--lighten-2">
+                User is active <span v-if="isActive">(currently active)</span
+                ><span v-else>(currently not active)</span>
+              </div>
+              <v-checkbox v-model="isActive" label="Is Active"></v-checkbox>
+              <v-row align="center">
+                <v-col class="flex-shrink-1">
+                  <v-checkbox v-model="setPassword" class="mr-2"></v-checkbox>
```

Comment:
A label would be nice here, so you know for what the button ist.
For Example:
```vue
                  <v-checkbox
                    v-model="setPassword"
                    class="mr-2"
                    label="Override Password"
                  ></v-checkbox>
```

Reply:
I think this is a change that would generally be beneficial to the main branch as well. I don't want to merge further improvements into this PR, which might not end up in main after all.

Reply:
I disagree with that. Even if this would be changed on the master, merging your pull request would remove this again. Therefore I'd like to include this right here and maybe add extra pull request to the master for this.

If you want to, you can accept this here or wait for the merge and change it later:
```suggestion
                  <v-checkbox
                    v-model="setPassword"
                    class="mr-2"
                    label="Override Password"
                  ></v-checkbox>
```

Reply:
Even if that is the case, it might be worth it to do a separate PR which I could merge into my branch as well. So if either gets accepted, The changes will be included. It might be redundant, but this way we can make sure everybody will benefit from these improvements eventually.

Nonetheless, I will accept these now, but would be happy if you write a separate PR as well. Small changes like this are easily reviewed and probably quick to merge.